aws/src/main/java/io/confluent/csid/config/provider/aws/AppendSecretPrefixRequestHandler2.java

```diff
@@ -1,137 +0,0 @@
- *    ANY KIND, INCLUDING BUT NOT LIMITED TO, LOST PROFITS OR ANY CONSEQUENTIAL,
- *    SPECIAL, INCIDENTAL, INDIRECT, OR DIRECT DAMAGES, HOWEVER CAUSED AND ON ANY
- *    THEORY OF LIABILITY, ARISING OUT OF THIS AGREEMENT.  THE FOREGOING SHALL
- *    APPLY TO THE EXTENT PERMITTED BY APPLICABLE LAW.
- *
- *    6.GENERAL.
- *
- *       6.1 Governing Law. This Agreement will be governed by and interpreted in
- *       accordance with the laws of the state of California, without reference to
- *       its conflict of laws principles.  If Licensee is located within the
- *       United States, all disputes arising out of this Agreement are subject to
- *       the exclusive jurisdiction of courts located in Santa Clara County,
- *       California. USA.  If Licensee is located outside of the United States,
- *       any dispute, controversy or claim arising out of or relating to this
- *       Agreement will be referred to and finally determined by arbitration in
- *       accordance with the JAMS International Arbitration Rules.  The tribunal
- *       will consist of one arbitrator.  The place of arbitration will be Palo
- *       Alto, California. The language to be used in the arbitral proceedings
- *       will be English.  Judgment upon the award rendered by the arbitrator may
- *       be entered in any court having jurisdiction thereof.
- *
- *       6.2 Assignment.  Licensee is not authorized to assign its rights under
- *       this Agreement to any third party. Confluent may freely assign its rights
- *       under this Agreement to any third party.
- *
- *       6.3 Other.  This Agreement is the entire agreement between the parties
- *       regarding the subject matter hereof.  No amendment or modification of
- *       this Agreement will be valid or binding upon the parties unless made in
- *       writing and signed by the duly authorized representatives of both
- *       parties.  In the event that any provision, including without limitation
- *       any condition, of this Agreement is held to be unenforceable, this
- *       Agreement and all licenses and rights granted hereunder will immediately
- *       terminate.  Waiver by Confluent of a breach of any provision of this
- *       Agreement or the failure by Confluent to exercise any right hereunder
- *       will not be construed as a waiver of any subsequent breach of that right
- *       or as a waiver of any other right.
- */
-package io.confluent.csid.config.provider.aws;
-
-import com.amazonaws.AmazonWebServiceRequest;
-import com.amazonaws.handlers.RequestHandler2;
-import com.amazonaws.services.secretsmanager.model.GetSecretValueRequest;
-
-class AppendSecretPrefixRequestHandler2 extends RequestHandler2 {
-  private final String prefix;
-
-  public AppendSecretPrefixRequestHandler2(String prefix) {
-    this.prefix = prefix;
-  }
-
-  @Override
-  public AmazonWebServiceRequest beforeExecution(AmazonWebServiceRequest request) {
```

Comment:
This class usage has been updated to use https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/core/interceptor/ExecutionInterceptor.html instead but using this requires the change to happen on the client rather than on the builder; but I have replicated the functionality elsewhere and tested for that via IT